Adapt kernel option menu to offer these options:

* default (quiet systemd.show_status=1)
* "MD5SUM" (quiet systemd.show_status=1 md5sum)
* "No Quiet" ()
* "To Ram" (quiet systemd.show_status=1 toram)
* "From ISO" (quiet systemd.show_status=1 fromiso)
* "Persist" (quiet systemd.show_status=1 persist)
* txt_kernel_safe (apm=off mce=off barrier=off ide=nodma idewait=50 i8042.nomux psmouse.proto=bare irqpoll pci=nommconf)
* txt_kernel_nolapic (nolapic)

© 2009, Niall Walsh <niallwalsh@users.berlios.de>
© 2014, Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/themes/openSUSE/src/dia_kernelopts.inc
+++ b/themes/openSUSE/src/dia_kernelopts.inc
@@ -11,9 +11,13 @@
 /kernelopts.default 0 def
 
 /kernelopts.options [
+  "quiet systemd.show_status=1"
+  "quiet systemd.show_status=1 md5sum"
   ""
-  "apm=off acpi=off mce=off barrier=off ide=nodma idewait=50 i8042.nomux psmouse.proto=bare irqpoll pci=nommconf"
-  "acpi=off"
+  "quiet systemd.show_status=1 toram"
+  "quiet systemd.show_status=1 fromiso"
+  "quiet systemd.show_status=1 persist"
+  "apm=off mce=off barrier=off ide=nodma idewait=50 i8042.nomux psmouse.proto=bare irqpoll pci=nommconf"
   "nolapic"
 ] def
 
@@ -31,8 +35,12 @@
   xmenu .xm_current kernelopts.default put
   xmenu .xm_list [
     /txt_kernel_default
+    "MD5SUM\n"
+    "No Quiet\n"
+    "To Ram\n"
+    "From ISO\n"
+    "Persist\n"
     /txt_kernel_safe
-    /txt_kernel_noacpi
     /txt_kernel_nolapic
   ] put
   xmenu .xm_title /txt_kernel put
